Add new english blog post (#130)
vivliostyle/vivliostyle.org: _posts/2022-11-30-event-report-of-Vivliostyle-user-dev-meetup-autumn-2022.md

---
title: Event report of “Vivliostyle User/Dev Meetup Autumn 2022”
lang: en
image: /assets/posts/2022-11-30-event-report-of-Vivliostyle-user-dev-meetup-autumn-2022/fig-1.png
author:  ogwata
tags:
  - event
---
<div style="float: right; margin: 0 0 1em 1em;"><img src="/assets/posts/2022-11-30-event-report-of-Vivliostyle-user-dev-meetup-autumn-2022/fig-1.png" alt="Event report of “Vivliostyle User/Dev Meetup Autumn 2022”" style="width: 500px;" /></div>

(The original article is [「Vivliostyle ユーザーと開発者の集い 2022秋」開催報告](https://vivliostyle.org/ja/blog/2022/11/30/event-report-of-Vivliostyle-user-dev-meetup-autumn-2022/) in Japanese)

## “Vivliostyle User/Dev Meetup Autumn 2022” was held

The annual fall user event was held online on November 20.

- [Vivliostyle User/Dev Meetup Autumn 2022](https://connpass.com/event/264332/)

The program for the day and links to the respective slides and archived videos are below.

- **Part 1**
    - 13:00-13:10（10 min.）**Explanation of the purpose of today's meeting**（[Katsuhiro Ogata](https://twitter.com/ogwata)）（[movie](https://youtu.be/g41n3xi86uQ)）
        - Here is a brief overview of the abstracts scheduled for presentation on this day.

>     - 13:10-13:40（30 min.）**CSS text-spacing property from the perspective of the preceding implementation of Vivliostyle.js**（[Suzuki Takeru, FONT WORKS](https://twitter.com/terkel)）（[slide](https://speakerdeck.com/terkel/vivliostyle-dot-js-noxian-xing-shi-zhuang-karakao-erucss-text-spacing-puropatei), [movie](https://youtu.be/vfb-FBTyAUU)）

Reviewer: ```suggestion
    - 13:10-13:40（30 min.）**CSS text-spacing property from the perspective of the preliminary implementation in Vivliostyle.js**（[Suzuki Takeru, Fontworks](https://twitter.com/terkel)）（[slide](https://speakerdeck.com/terkel/vivliostyle-dot-js-noxian-xing-shi-zhuang-karakao-erucss-text-spacing-puropatei), [movie](https://youtu.be/vfb-FBTyAUU)）
```

>         - We will review the prior implementation of Vivliostyle.js (spacing between Japanese and Western words, spacing of consecutive punctuation, and spacing at the beginning and end of lines), which has not yet been implemented in the browser, from the user's perspective. From the last 10 minutes, Representative Murakami, the implementer, will answer questions.

Reviewer: ```suggestion
        - Reviews the preliminary implementation in Vivliostyle.js (spacing between Japanese and Western words, spacing of consecutive punctuation, and spacing at the beginning and end of lines), which has not yet been implemented in browsers, from the user's perspective. From the last 10 minutes, Representative Murakami, the implementer, answers questions.
```
